docs/getting-started/github-apps/github-apps.rst

```diff
@@ -46,14 +46,12 @@ How the Dockstore GitHub App works
 
 With the Dockstore GitHub App installed, the synchronization is done automatically. When
 you add a new branch or release of a workflow on GitHub, Dockstore is notified,
-and Dockstore updates its copy of the workflow. For example, After publishing a new release
-of a workflow on GitHub, a new version of the workflow will be present in
-Dockstore shortly afterwards. For this to work, a .dockstore.yml file is **required** in the root directory (or inside the ``.github`` directory) of each GitHub repository you want
+and Dockstore updates its copy of the workflow. For example, after publishing a new release
+of a workflow on GitHub, a new version of the workflow will be present in Dockstore shortly afterwards. For this to work, a .dockstore.yml file is **required** in the root directory (or inside the ``.github`` directory) of each GitHub repository you want
```

Comment:
Instead of twice using the parens with exceptions for the .github directory, which

1. Makes it harder to read (I say as an abuser myself of parenthetical asides)
2. Makes it seem like an exception/oddity to have it in the `.github` directory, when arguably that is the better place to put it.

I would just say ` ... .dockstore.yml file is required in either the root directory or in the .github directory...`, and entirely remove the the `It should not be inside any subfolder (except ``.github``)` sentence.

Reply:
I've given this some thought and ultimately decided against it, as I have seen people interpret "the root directory" to mean "any subfolder of the root directory."